include/filefuncs.h

```diff
@@ -28,7 +28,7 @@ const char *BaseName(const char *p);
 Bool FileExists(const char *f);
 Bool DirExists(const char *d);
 Bool ChDir(const char *d);
-Bool MkDir(const char *d);
+Bool MkDir(const char *dname, LOG_INFO* LogInfo);
 Bool RemoveFiles(const char *fspec, LOG_INFO* LogInfo);
 void sw_error(int errorcode, const char *format, ...);
```

Comment:
We can make it a private, non-static function. The compiler flag `[-Wunused-function]` has an issue when it is static and not used (since we will only use it in STEPWAT2).

Reply:
See my other comment about removing `sw_error()` -- if not then we should wrap `static void sw_error()` in `#if defined (STEPWAT)` directives - exactly because it is unused in SOILWAT code